shared/protocol/src/test/java/io/pravega/shared/protocol/netty/AppendEncodeDecodeTest.java

```diff
@@ -316,7 +318,7 @@ public void testAppendDecoderInvalidAppendBlockEnd() throws Exception {
     }
 
     @Test(expected = IllegalArgumentException.class)
```

Comment:
For all the tests in this file let's set the leak checker to `Paranoid`. As that will ensure that we don't leak any byte buffers in this code.

Reply:
It is paranoid already.